src/widgets/table.rs

```diff
@@ -286,14 +289,21 @@ impl<'a> Table<'a> {
         self
     }
 
+    /// If set to `true`, the selection gap will always be allocated, regardless of if something is
+    /// selected or not
+    pub fn highlight_set_always_space(mut self, value: bool) -> Self {
+        self.highlight_always_space = value;
+        self
+    }
+
     pub fn column_spacing(mut self, spacing: u16) -> Self {
         self.column_spacing = spacing;
         self
     }
 
-    fn get_columns_widths(&self, max_width: u16, has_selection: bool) -> Vec<u16> {
+    fn get_columns_widths(&self, max_width: u16, add_selection_space: bool) -> Vec<u16> {
```

Comment:
done with 21a7869ccd5e3380a0cd82826f1044ecd528e352, but had to do e2a53e1d1e3fcb543f60e8a60b26b48023a96f74 before, otherwise it would not made have proper sense